packages/react/src/section-alert/SectionAlert.tsx

```diff
@@ -58,34 +60,50 @@ export const SectionAlert = forwardRef<HTMLDivElement, SectionAlertProps>(
 			focusOnUpdate,
 			forwardedRef,
 		});
+		const { childrenId, titleId, toneId } = useSectionAlertIds(id);
 
-		const Icon = sectionAlertIconMap[tone];
+		const icon = sectionAlertIconMap[tone];
 		const closeHandler = getOptionalCloseHandler(onClose, onDismiss);
 
 		return (
 			<Flex
+				{...props}
 				alignItems="center"
+				aria-labelledby={`${toneId} ${titleId} ${children ? childrenId : ''}`}
```

Comment:
Wanted to use `aria-describedby` for `childrenId`, but that made things worse in NVDA (the first one I tested), so stopped trying that